src/js/components/Header/VolumeControl.react.js

```diff
@@ -32,7 +32,7 @@ export default class VolumeControl extends Component {
     }
 
     getVolumeIcon(volume, muted) {
-        return muted || volume === 0 ? 'volume-off' : volume > 0.5 ? 'volume-up' : 'volume-down';
+        return muted || volume === 0 ? 'volume-off' : Math.pow(volume, 1 / 4) > 0.5 ? 'volume-up' : 'volume-down';
```

Comment:
You don't see any problem if `LinearizeVolume` is in the component file as we're only using it here right ?